Terminal.Gui/Views/ScrollBarView.cs

```diff
@@ -109,6 +116,34 @@ public ScrollBarView (View host, bool isVertical, bool showBothScrollIndicator =
 			contentBottomRightCorner.MouseClick += ContentBottomRightCorner_MouseClick;
 		}
 
+		private void Host_VisibleChanged ()
+		{
+			if (!Host.Visible) {
+				Visible = Host.Visible;
+				if (otherScrollBarView != null) {
+					otherScrollBarView.Visible = Visible;
+				}
```

Comment:
Should this code be in `ShowHideScrollBars` instead?  It looks like there is already logic in there for setting visibility aspects correctly based on a verity of conditions.  Are there corner cases where having this logic in `ShowHideScrollBars` would be safer (e.g. adding scrollbars to a view that is already Visible=false to begin with)

Reply:
I tried that before but there was no way to distinguish if Visible is false settled by the ShowHideScrollBars or settled by the host.

> (e.g. adding scrollbars to a view that is already Visible=false to begin with)

I think in the constructor is more safer.

Reply:
@tznind you were right. Please be at ease if you do not agree with my justification and yet you think you are right, because sometimes I do not see the problem of beginning. Done. Thanks.